go/tasks/plugins/presto/config/config.go

```diff
@@ -0,0 +1,78 @@
+package config
+
+//go:generate pflags Config --default-var=defaultConfig
+
+import (
+	"context"
+	"net/url"
+	"time"
+
+	"github.com/lyft/flytestdlib/config"
+	"github.com/lyft/flytestdlib/logger"
+
+	pluginsConfig "github.com/lyft/flyteplugins/go/tasks/config"
+)
+
+const prestoConfigSectionKey = "presto"
+
+func URLMustParse(s string) config.URL {
```

Comment:
lower case? 
```suggestion
func urlMustParse(s string) config.URL {
```

Reply:
I made it  public because I call the method in one of the unit test

Reply:
I like URL.